### ISSUE TITLE
Cart is not accessible to VoiceOver in base styles

### ISSUE THREAD

**Issue by scottsb** [magento/magento2#21447]
### Preconditions (*)
All versions from 2.0.0 to 2.3.0

### Steps to reproduce (*)
1. Add two items to cart.
2. Use VoiceOver on Mac to navigate through cart using "next item" keyboard shortcut or gesture.

### Expected result (*)
VoiceOver navigates through each cell in sequence.

### Actual result (*)
1. VoiceOver skips the first cell containing the item name, meaning that a visually impaired user cannot understand what is actually in their cart.
2. VoiceOver skips the last row in the cart (containing action buttons), meaning they cannot remove or edit the last item from their cart.

### Diagnosis
These have two separate causes:
1. VoiceOver skips the first table cell because it is marked with `display: block`, which VoiceOver interprets as meaning the cell is only presentational. Anytime CSS changes the display property on a table element, it [effectively changes the implicit role as well](https://developer.paciellogroup.com/blog/2018/03/short-note-on-what-css-display-properties-do-to-table-semantics/). This can be fixed by either getting rid of the CSS rule, which doesn't seem necessary in the first place, or by explicitly adding a `role="gridcell"` to the first `<td>` where the CSS rule is applied.
2. VoiceOver skips the last row due to how the `<caption>` is being hidden. This seems like it may be a bug in VoiceOver itself, but by virtue of the `position: absolute` being applied through the `lib-visually-hidden` mixin, it seems to remove it from the table (for a similar reason as the prior point), yet fail to internally account for this properly when determining how many rows to read through.

I may submit a PR for this, but I'm waiting for feedback on the best way to resolve the second issue. Glad to hear any suggestions here, and I've also posted on Stack Overflow:
https://stackoverflow.com/questions/54876969/how-to-accessibly-hide-table-caption-element

**Comment by magento-engcom-team** [comment]
Hi @scottsb. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@scottsb do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by Jitheesh** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Jitheesh. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Jitheesh, here is your [Magento instance](https://i-21447-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21447-2-3-develop.instances.magento-community.engineering/admin](https://i-21447-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by adarshkhatri** [comment]
@magento-engcom-team give me 2.3.0 instance

**Comment by magento-engcom-team** [comment]
Hi @adarshkhatri. Thank you for your request. I'm working on Magento 2.3.0 instance for you

**Comment by magento-engcom-team** [comment]
Hi @adarshkhatri, here is your [Magento instance](https://i-21447-2-3-0.instances.magento-community.engineering).
Admin access: [https://i-21447-2-3-0.instances.magento-community.engineering/admin](https://i-21447-2-3-0.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by adarshkhatri** [comment]
@magento-engcom-team give me 2.3.0 instance

**Comment by magento-engcom-team** [comment]
Hi @adarshkhatri. Thank you for your request. I'm working on Magento 2.3.0 instance for you

**Comment by magento-engcom-team** [comment]
Hi @adarshkhatri, here is your [Magento instance](https://i-21447-2-3-0.instances.magento-community.engineering).
Admin access: [https://i-21447-2-3-0.instances.magento-community.engineering/admin](https://i-21447-2-3-0.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar, here is your [Magento instance](https://i-21447-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21447-2-3-develop.instances.magento-community.engineering/admin](https://i-21447-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98480`, `MAGETWO-98481` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by scottsb** [comment]
I do not have time to verify at the moment that this is still valid in 2.4; however, this is an important issue for WCAG compliance, so I don't want it to be closed automatically.

**Comment by dverkade** [comment]
Posting a comment. Issues should not be automatically closed.

**Comment by engcom-Bravo** [comment]
Hi @scottsb,

Thanks for your reporting and collaboration.

We have gone through with the description and it seems to be an enhancement to proceed further and marking this as **Feature Request**

Thanks.

**Comment by engcom-Bravo** [comment]
HI @scottsb,

Thanks for your reporting and collaboration.

We have tried to reproduce the issue in Latest 2.4-develop instance and the issue is no more reproducible.Hence we are closing this issue.

Thanks.
